tools/ds-identify

```diff
@@ -179,10 +179,30 @@ debug() {
     echo "$@" 1>&3
 }
 
+map_sysfs_to_dmi() {
+    case "$@" in
+        sys_vendor) _RET="system-manufacturer";;
+        product_name) _RET="system-product-name";;
+        product_uuid) _RET="system-uuid";;
+        product_serial) _RET="system-serial-number";;
+        chassis-asset-tag) _RET="chassis_asset_tag";;
+        *) return 1;;
+    esac
+}
+
 get_dmi_field() {
     local path="${PATH_SYS_CLASS_DMI_ID}/$1"
     if [ ! -f "$path" ] || [ ! -r "$path" ]; then
-        _RET="$UNAVAILABLE"
+        map_sysfs_to_dmi "$1" || {
```

Comment:
i think maybe put this code together with the map code above.

http://paste.ubuntu.com/p/BZqDpTdqQH/